packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx

```diff
@@ -159,7 +168,7 @@ export function CategoryAutocomplete({
   );
 
   return (
-    <Autocomplete
+    <Autocomplete<CategoryAutocompleteItem>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we need this? Previously the type could be inferred from  `items`